data/darktableconfig.xml.in

```diff
@@ -313,20 +313,32 @@
     <shortdescription>omit hierarchy in simple tag lists</shortdescription>
     <longdescription>when creating XMP file the hierarchical tags are also added as a simple list of non-hierarchical ones to make them visible to some other programs. when this option is checked darktable will only include their last part and ignore the rest. so 'foo|bar|baz' will only add 'baz'.</longdescription>
   </dtconfig>
-  <dtconfig prefs="misc" section="tags">
-    <name>plugins/lighttable/tagging/no_uncategorized</name>
-    <type>bool</type>
-    <default>false</default>
-    <shortdescription>in collect module do not set the 'uncategorized' entry for tags</shortdescription>
-    <longdescription>in collect module do not set the 'uncategorized' entry for tags which do not have children</longdescription>
-  </dtconfig>
   <dtconfig prefs="misc" section="tags" restart="true">
     <name>plugins/lighttable/tagging/no_entry_completion</name>
     <type>bool</type>
     <default>false</default>
     <shortdescription>disable the entry completion</shortdescription>
     <longdescription>the entry completion is useful for those who enter tags from keyboard only. for others the entry completion can be embarrassing. need to restart darktable.</longdescription>
   </dtconfig>
+  <dtconfig prefs="misc" section="tags">
+    <name>plugins/lighttable/tagging/no_uncategorized</name>
+    <type>bool</type>
+    <default>false</default>
+    <shortdescription>in collect module, do not set the 'uncategorized' entry for tags</shortdescription>
+    <longdescription>in collect module, do not set the 'uncategorized' entry for tags which do not have children</longdescription>
+  </dtconfig>
+  <dtconfig prefs="misc" section="tags">
+    <name>plugins/lighttable/tagging/case_sensitivity</name>
+    <type>
+      <enum>
+        <option>sensitive</option>
+        <option>insensitive</option>
+      </enum>
+    </type>
+    <default>insensitive</default>
+    <shortdescription>in collect module, tags case sensitivity</shortdescription>
+    <longdescription>in collect module, tags case sensitivity. without the Sqlite ICU extension, insensivity works only for English letters</longdescription>
```

Comment:
While working on the French translation I'm wondering if instead of:

**for English letters.**

We should not be useing

**for non accentuated latin-1 letters.**

What do you think?

Reply:
I'm not completely sure if "English letters" is the right wording, but "latin-1" norms are complex and that may not be clearer.

By English letters I meant the only 26 letters of English alphabet (upper and lower cases) which should make sense in common language.
To be more accurate maybe: "... for the 26 letters of English alphabet (without accent)" ?

Reply:
Maybe, fact is that English seems misleading here. We have the same 26 letters in French and probably other languages share the same letters. Maybe ASCII ?

Reply:
You don't like it... :smile:
the "26 ASCII letters" would be accurate enough. Or "26 latin letters". For me the point is to be understood by non geek guys and without introducing complex norms.

Reply:
:) I'll go with 21 latin letters

Reply:
21 ?

Reply:
A stupid typo :( Fixed now.